Update docs related to position to account for static (#4235)
facebook/react-native-website: docs/flexbox.md

</TabItem>
</Tabs>

## Position

The `position` type of an element defines how it is positioned relative to either itself, its parent, or its [containing block](./flexbox.md#the-containing-block).

- `relative` (**default value**) By default, an element is positioned relatively. This means an element is positioned according to the normal flow of the layout, and then offset relative to that position based on the values of `top`, `right`, `bottom`, and `left`. The offset does not affect the position of any sibling or parent elements.

- `absolute` When positioned absolutely, an element doesn't take part in the normal layout flow. It is instead laid out independent of its siblings. The position is determined based on the `top`, `right`, `bottom`, and `left` values. These values will posistion the element relative to its containing block.

> - `static` When positioned statically, an element is positioned according to the normal flow of layout, and will ignore the `top`, `right`, `bottom`, and `left` values. This `position` will also cause the element to not form a containing block, unless some other supersceding style prop is present (e.g. `transform`). This allows `absolute` elements to be positioned to something that is not their parent. Note that `static` is only available on the new architecture.

Reviewer: Could we add the caveat this is only supported in new arch?

Reviewer: This was also introduced a while enough back that we could add to the previous versioned copies of documents, but there has historically been push back on including new arch APIs before 0.76 where we enable by default.

> - `static` When positioned statically, an element is positioned according to the normal flow of layout, and will ignore the `top`, `right`, `bottom`, and `left` values. This `position` will also cause the element to not form a containing block, unless some other supersceding style prop is present (e.g. `transform`). This allows `absolute` elements to be positioned to something that is not their parent. Note that `static` is only available on the new architecture.

Reviewer 2: ```suggestion
- `static` When positioned statically, an element is positioned according to the normal flow of layout, and will ignore the `top`, `right`, `bottom`, and `left` values. This `position` will also cause the element to not form a containing block, unless some other superceding style prop is present (e.g. `transform`). This allows `absolute` elements to be positioned to something that is not their parent. Note that **`static` is only available on the New Architecture**.
```
When we call out the New Architecture, we always use capital letters :wink:

</TabItem>
</Tabs>

## The Containing Block

The containing block of an element is an ancestor element which controls its position and size.
The way containing blocks work in React Native is very similar to [how they work on the web](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block), with some simplifications due to the lack of some web features.

The `top`, `right`, `bottom`, and `left` values of an absolutely positioned element will be
relative to its containing block.

> Percentage lengths like `width: '50%'` or `padding: '10%'` applied to absolutely positioned elements will be calculated relative to the size of its containing block. So for example, if the containing block is 100 points wide, then `width: 50%` on an absolutely positioned element will cause it to be

Reviewer 2: ```suggestion
Percentage lengths (e.g.: `width: '50%'` or `padding: '10%'`) applied to absolutely positioned elements will be calculated relatively to the size of its containing block. For example, if the containing block is 100 points wide, then `width: 50%` on an absolutely positioned element will cause it to be 50 points wide.
```

> 50 points wide.

Reviewer 2: ```suggestion

```

Why this was on a new line? 🤔

Author: oops, just a typo
